src/protocols/LoRaWAN/LoRaWAN.h

```diff
@@ -409,21 +466,57 @@ class LoRaWANNode {
     */
     void setDeviceStatus(uint8_t battLevel);
 
+    /*!
+      \brief Set uplink datarate. This should _not_ be used when ADR is enabled.
+      \param dr Datarate to use for uplinks
+      \returns \ref status_codes
+    */
+    int16_t setDatarate(uint8_t drUp);
+
+    /*!
+      \brief Toggle ADR to on or off
+      \param enable Whether to disable ADR or not
+    */
+    void setADR(bool enable = true);
+
+    /*!
+      \brief Select a single subband (8 channels) for fixed bands such as US915
+      \param idx The subband to be used (starting from 1!)
```

Comment:
Why is this indexed from 1?

Reply:
TTN names the subbands starting from 1, and as that is my testing ground, I copied that. Not sure how Helium does this.

Reply:
Judging from [this](https://docs.helium.com/iot/lorawan-frequency-plans/#au915_sb1) Helium also starts from 1